CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 1.4.30-M1 (Dev 2020)
+  * [KT-43597](https://youtrack.jetbrains.com/issue/KT-43597) Xcode 12.2 support
+  * [KT-43276](https://youtrack.jetbrains.com/issue/KT-43276) Add watchos_x64 target
+  * Support value classes
```

Comment:
* Significant improvement of compilation time.

Reply:
@homuroll Do we have a KT issue to reference to? Or at least PR/commits?